.github/workflows/on_pull_request_open.yml

```diff
@@ -2,16 +2,39 @@ name: Opened Pull Request Labeler
 
 on:
   pull_request:
-    types: [opened]
+    types: [opened, synchronize]
 
 jobs:
-  pr-triage:
+  pr-context-labeller:
```

Comment:
labeller vs labeler ? this is one of those words that I can never actually remember how to spell lol. I think it's labeler